app/assets/stylesheets/budgets/groups_and_headings.scss

```diff
@@ -5,16 +5,22 @@
   justify-content: space-between;
   width: 100%;
   margin-bottom: 30px;
+  row-gap: 30px;
```

Comment:
@andreiio Feel free to ignore this warning; the linter we're using doesn't recognize this property and we don't use it in CONSUL because [it isn't universally supported yet](https://caniuse.com/mdn-css_properties_row-gap_flex_context), but I see the browser support in Romania seems to be higher than 95%, so this shouldn't be an issue here :tada:.

Reply:
got it, thanks for looking into it!